spark-hs/templates/spark-configmap.yaml

```diff
@@ -10,7 +10,9 @@ data:
     {{- if eq $schema "amazon" }}
     spark.hadoop.fs.s3a.access.key=$AWS_ACCESS_KEY_ID
     spark.hadoop.fs.s3a.secret.key=$AWS_SECRET_ACCESS_KEY
-
+    {{- if ne .Values.sparkEventLogStorage.endpoint "" }}
```

Comment:
Could you change to `{{- if  .Values.sparkEventLogStorage.endpoint  }}` ?

Reply:
done!